clientd3d/d3drender_world.c

```diff
@@ -40,10 +40,15 @@ extern int gNumObjects;
 extern Vector3D sun_vect;
 extern D3DMATRIX view;
 
-static const auto alpha_test_threshold = 128; // Threshold for alpha testing to determine transparency
-extern Bool gWireframe;
+// The alpha_test_threshold value of 200 was chosen based on tuning through experimentation.
```

Comment:
I'm still confused by this.  An alpha threshold doesn't seem like a tuning parameter---I would think that one value is correct, and all other values are wrong.  What does this value actually do?